Application/Api/Billing/Bookkeeping/ApiBasket.php

```diff
@@ -118,12 +118,13 @@ public static function receiverService($Content = '')
 
     /**
      * @param string $Identifier
+     * @param string $Type
      * @param array  $Basket
      * @param array  $ErrorHelp
      *
      * @return Pipeline
      */
-    public static function pipelineOpenAddBasketModal($Identifier = '', $Basket = array(), $ErrorHelp = array())
+    public static function pipelineOpenAddBasketModal($Identifier = '', $Type = '', $Basket = array(), $ErrorHelp = array())
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Identifier" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)